packages/metadata/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@jspsych/metadata",
+  "version": "0.0.1",
+  "description": "jsPsych plugin for creating and customizing metadata according to Psych-DS standards.",
```

Comment:
I don't think this is a "plugin" in the traditional jsPsych sense.

Reply:
Changed to "package".